x11-wm/fvwm3/fvwm3-1.0.8.ebuild

```diff
@@ -92,6 +92,10 @@ RDEPEND="${PYTHON_DEPS}
 DEPEND="${COMMON_DEPEND}
 	x11-base/xorg-proto"
 
+PATCHES=(
+	"${FILESDIR}/${P}-implicit-function-decl-configure.patch"
+)
+
```

Comment:
I noticed -9999 suffers from this issue, but no need to patch it - hopefully will get merged upstream.

Reply:
I'm assuming so. Thanks for the merge!